conduct.md

```diff
@@ -7,16 +7,282 @@
 
 ## Categories of Conduct
 
+* Conduct
 * Attendance
 * Deadlines
 * Communication
 * Participation
-* Conduct
 * Decisions
 * Postmortems
 * Conflicts
 * Infractions
 
+## Attendance
+
+It is expected that you will attend all Labs on time. If you and
+your team deem it possible, team members may leave the lab, so long as they
+are available via `Slack` for the allotted lab times. This would be considered
+working remotely, and requires the consent of the other team members or
+approval from the instructor. For further guidance on attendance in `Labs` and
+`Class` please refer to the [course
+website](https://www.gregorykapfhammer.com/teaching/cs203S2019/), contact the
+[instructor](https://www.gregorykapfhammer.com/contact/), or schedule an
+[appointment](https://www.gregorykapfhammer.com/schedule/).
+
+* **Acceptable Attendance (C):**
+  * Attending all lab sessions on time
+  * Missing sessions with consent of the course instructor
+  * Leaving labs early with consent of teammates or the course instructor
+  * Being available on Slack during allotted lab times
+
+* **Unacceptable Attendance (U):**
+  * Missing a lab session without consent of the course instructor
+  * Arriving more than 10 minutes late to a lab session
+  * Leaving labs early while still needed by teammates
+
+## Conduct
+
+In this context, we will be defining conduct as the way a team member behaves
+and the way they act when carrying out their part of the project. First, everyone
+is obligated to adhere to the 9 categories in the Code of Conduct. They will be
+our foundation throughout the semester when working together. Moving on, everyone
+should respect one another; no one should be excluded based on their race or gender.
+Team members should not shun the ideas of anyone, especially those who may
+have not have had an equal amount of experience as others. Also, at the beginning
+of each session, everyone should enter with positive attitudes, These positive
+attitudes should stay consistent whether the project is working or breaking.
+
+* **Acceptable Conduct (A):**
+  * Respect to all peers
+  * Allowing all members to voice their opinion
+  * Showing positive attitudes towards all aspects of the project
+
+* **Unacceptable Conduct (U):**
+  * Excluding members of team based on their race or gender
+  * Refusal to follow the rest of the categories in the Code of Conduct
+  * Showing a negative attitude towards group members or aspects of the project
+
+## Deadlines
```

Comment:
The formatting of bullets and indentation in this section does not match the rest of document.

Reply:
```
* **Unacceptable Conduct (U):**
  * Excluding members of team based on their race or gender
  * Refusal to follow the rest of the categories in the Code of Conduct
  * Showing a negative attitude towards group members or aspects of the project
```

did a git pull and this is what we are seeing in atom. all elements seem to be on the appropriate line now

Reply:
@Michionlion this problem has been fixed